README.md

```diff
@@ -79,6 +79,17 @@ Must be deployed as a Kubernetes **Deployment**. Also requires some **additional
 |        ASG Termination Lifecycle Hooks        |       ❌        |        ✅        |
 |         Instance State Change Events          |       ❌        |        ✅        |
 
+### Kubernetes Compatibility
+
+|  NTH Release  | K8s v1.26 | K8s v1.25 | K8s v1.24 | K8s v1.23 | K8s v1.22 | K8s v1.21 | 
+| :-----------: | :-------: | :-------: | :-------: | :-------: | :-------: | :-------: | 
+|    v1.20.0    |     ✅    |    ✅     |    ✅     |    ✅     |    ✅     |    ❌    |
```

Comment:
Good work @GavinBurris42 on your first PR 👍 , small suggestion could you make these NTH release versions as hyperlinks, thanks!